### PR TITLE
Html css test

### PR DESCRIPTION
**Pull request by mirwan918**
- Add HTML file and content
- Add CSS file and CSS style
- Add linters 
- Add .gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+ 
+ ```
```
Comment (topeogunleye) [inline README.md:83]: - [ ] In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
  - The only optional section is FAQ.
  - Kindly add the Future Features section.

- [ ] I kindly suggest that you delete this `gem install node js` kindly add the proper steps for running your project locally

For Example, you could add an instructions on how to run your projects locally:

## EXAMPLE

```
## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Prerequisites
Basic knowledge of HTML, CSS, and JavaScript.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/mirwan918/Hello-Microverse-Project/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:
```

```bash
git clone git@github.com:mirwan918/Hello-Microverse-Project.git
```

```
- Once the files are on your machine, open the _portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

### Install

The following tools help make easier to work with sample code.

- [git](https://git-scm.com/downloads): A tool for managing source code
- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
- [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer): A simple web server utility for Visual Studio Code

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @mirwan918, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please kindly add a more descriptive PR title and description as seen in the demo below:
pull_request_desc

- [ ] You used the correct GitHub flow, but the branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). Kindly rename 'html-css-test' following [this article](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) or checkout to a new branch and use the correct name and mention the old PR in the new PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mirwan918) [reply README.md:83]: @topeogunleye  Why are future features important? Is there a central guideline on how to a README?

**Comment by mirwan918** [comment]
@topeogunleye  This is in regards to pull request  changes. After renaming the branch this pull request closed. How do I share the pull request changes using the same PR link?

File: README.md (hunk 1 above)
Comment (mirwan918) [reply README.md:83]: @topeogunleye Kindly is there a youtube video that relates to the above recomendations ? This is will help further to understand in laymans language. Also I have researched and I haven't come across similar content.

Comment (topeogunleye) [reply README.md:83]: > @topeogunleye Why are future features important? Is there a central guideline on how to a README?

@mirwan918 Yes, the central guideline is from Microverse.

Comment (topeogunleye) [reply README.md:83]: > @topeogunleye Kindly is there a youtube video that relates to the above recomendations ? This is will help further to understand in laymans language. Also I have researched and I haven't come across similar content.

@mirwan918, It's a suggestion and an example. You are free to write yours differently.

**Comment by topeogunleye** [comment]
> @topeogunleye This is in regards to pull request changes. After renaming the branch this pull request closed. How do I share the pull request changes using the same PR link?

@mirwan918, Kindly add a link to the old PR in your new PR description

File: README.md (hunk 1 above)
Comment (topeogunleye) [reply README.md:83]: @mirwan918, Future features are important because when recruiters come across your README, they'll see it and be impressed 😏

Comment (topeogunleye) [reply README.md:83]: > @topeogunleye Kindly is there a youtube video that relates to the above recomendations ? This is will help further to understand in laymans language. Also I have researched and I haven't come across similar content.

@mirwan918, I don't know of any YouTube videos. You can checkout your favorite repo's README for some inspiration.

**Comment by mirwan918** [comment]
@topeogunleye  Apologies it is still not clear. Please see the screenshot below. Is this what you meant?
Screenshot (384)

**Comment by topeogunleye** [comment]
> @topeogunleye Apologies it is still not clear. Please see the screenshot below. Is this what you meant? Screenshot (384)

Sorry for the late reply. Yes @mirwan918
